R/chartutils.R

```diff
@@ -1222,22 +1225,27 @@ setTicks <- function(minimum, maximum, distance, reversed = FALSE,
     if (!is.null(data))
     {
         is.bar <- grepl("Bar", type) && !grepl("Stacked", type)
-        if (is.null(minimum))
-            minimum <- min(0, min(data, na.rm = TRUE))
-        if (is.null(maximum))
-            maximum <- max(data, na.rm = TRUE)
+        dat_min <- min(0, data, na.rm = TRUE)
```

Comment:
`dat.min` instead of `dat_min`.

Reply:
Renamed in [a52e9a1](https://github.com/Displayr/flipStandardCharts/pull/117/commits/a52e9a1562d93699dd331f6ce6ba6311df3745f4)